sources/core/Stride.Core.Mathematics/Int3.cs

```diff
@@ -143,6 +143,16 @@ public Int3(int[] values)
             Z = values[2];
         }
 
+        /// <summary>
+        /// Creates a new instance of the <see cref="Int3"/> struct by rounding the X, Y, and Z components of a <see cref="Vector3"/> struct to the nearest integer.
+        /// </summary>
+        public Int3 RoundToInt3(Vector3 value)
```

Comment:
Can you make this a constructor or an explicit cast instead? And instead of using `Math.Round` I think it would be better to use explicit cast on it like the constructor with `Vector2`
```
public static explicit operator Int3(Vector3 value)
{
    return new Int3((int)value.X, (int)value.Y, (int)value.Z);
}
```

Reply:
Casting to int is not the same as rounding, casting to int removes the fractional part of the value (be it positive or negative numbers), while rounding, well rounds up or down depending on the fractional part.
If the method is called `Round` it should be rounding, and that's a very particular operation, I do think it should be a specific function instead of a constructor.